pandas/core/tools/timedeltas.py

```diff
@@ -18,21 +18,27 @@
 
 def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     """
-    Convert argument to timedelta
+    Convert argument to timedelta.
+
+    Timedeltas are differences in times, expressed in difference units,
+    for example, years, momths, days, hours, minutes, seconds.
+    They can be both positive and negative. This method can create Timedelta
```

Comment:
this is not true at all

only Days and smaller are supoorted 

these are fixed time units

if you want variable units use an Offset which are designed for this